services/harmony/app/backends/workflow-orchestration/work-item-updates.ts

```diff
@@ -421,6 +421,35 @@ async function maybeQueueQueryCmrWorkItem(
   }
 }
 
+
+/**
+ * Check to see if a workflow step for the given job is completed
+ *
+ * @param logger - the logger to use for logging timing statements
+ * @param tx - the database transaction to use for making queries
+ * @param workItem - the work-item o
+ */
+async function isStepComplete(logger: Logger, tx: Transaction, workItem: WorkItem, thisStep: WorkflowStep): Promise<boolean> {
+  let notComplete = await (await logAsyncExecutionTime(
+    isDefinitelyNotComplete,
+    'HWIJUWJI.isDefinitelyNotComplete',
+    logger))(
+    tx, workItem.jobID, workItem.workflowStepIndex,
+  );
+
+  if (!notComplete) {
```

Comment:
if we negated the other function this would be `if (possiblyComplete)` - I can understand `!notComplete` so think it's good to leave it this way

Reply:
ok